src/sync/incomingSyncUtils.js

```diff
@@ -353,8 +360,10 @@ export function createOrUpdateRecord(database, settings, recordType, record) {
         status: STATUSES.translate(record.status, EXTERNAL_TO_INTERNAL),
         confirmDate: parseDate(record.confirm_date),
         theirRef: record.their_ref,
+        linkedRequisition,
       };
       const transaction = database.update(recordType, internalRecord);
+      if (linkedRequisition) linkedRequisition.linkedTransaction = transaction;
```

Comment:
Can we please stop thinking about .save and other notifiers as related to sync. They call listeners, and any part of the app may want to know about the change of a given data type, so we just have to consistently make sure we call .save any time we use dot notation. If that's too hard, we will have to rule out dot notation all together, and always just use database.update.

To talk specifically to your point about it not being synced here, well at the point you are in incomingSyncUtils you are using SyncDatabase anyway, so any changes will not be added to the SyncQueue (as they will have `causedBy === 'sync'`)